src/graphic/image_cache.cc

```diff
@@ -42,13 +42,24 @@ const Image* ImageCache::insert(const std::string& hash, std::unique_ptr<const I
 void ImageCache::fill_with_texture_atlases(
    std::vector<std::unique_ptr<Texture>> texture_atlases,
    std::map<std::string, std::unique_ptr<Texture>> textures_in_atlas) {
+	// There may still be pointers to existing images and textures, so we need
+	// to keep them around but no longer return any new pointers to them.
+
+	for (auto& t : texture_atlases_) {
+		outdated_texture_atlases_.push_back(std::move(t));
+	}
```

Comment:
This doesn't look right to me, but it works...
If I imagine to reload the atlas over and over again, this results in memory waste?
Maybe add a todo for the future to fix the pointers